### ISSUE TITLE
Move the grouping "annotations" whith its two settings from advanced settings to document formating settings category

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#15465]
### Is your feature request related to a problem? Please describe.
In browsers it is possible for NVDA to report "has details" on structured annotations and also to report Aria description. This feature has been introduced some whole ago and it is not in the testing phase anymore. It is good that the user has the choice whether to hear this or not.

### Describe the solution you'd like
Move the grouping together with the two checkboxes from advanced settings to document formating settings, and rename the grouping to "annotations in virtual documents".

### Describe alternatives you've considered
Leave as is but this is confusing for the users and the user experience is not intuitive.

**Comment by CyrilleB79** [comment]
For anyone wanting to work on this:

1. Do not use the expression "virtual documents"."; this concept is not known and it appears nowhere in the user guide. Only the browse mode caret / cursor is qualified "virtual" in the user guide.

2. The document formatting settings panel has a lot of options. Special care should be given to the GUI design, labels and UX so that the organisation of this panel remains as clear as possible.

**Comment by Nael-Sayegh** [comment]
Hello,

What would be the appropriate name for this group in the settings panel?

**Comment by Adriani90** [comment]
Actually I would say it is ok to move those check boxes from the annotations grouping directly to the grouping "elements" in the document formating settings, because this affects only web elements. So there is no separate grouping needed for the two settings.

**Comment by Adriani90** [comment]
This is similar to i.e. block quotes which is also part of the elements grouping.

**Comment by Adriani90** [comment]
I edited the issue description accordingly.

**Comment by CyrilleB79** [comment]
Frankly if these options are moved, it would be nice, for each possible configuration, to give:
* a use case where this config is useful
* an example where this config is useful.

These options are checked by default. And it does not seem to me that it causes any issue; I have not heard of anyone having issue with this either.

Though, it's true that I sometimes hear "has details", which is a useless information for me; indeed, if I press NVDA+D there, nothing happens, except on the example pages of W3C.

From what I understand these options are still experimental, and I do not know why this issue has been accepted. @seanbudd, clarification from your side would be needed since you have added the triaged label. Thanks.

Anyway, if the triaged status of this issue is confirmed, I'd recommend to make much more explicit the names of these options, as well as their description in the User Guide, not only for me but for many people.

**Comment by Adriani90** [comment]
@CyrilleB79 aria details replaces the old and deprecated long desc atribute which was mainly supported only in iE if I am not mistaken.
However, all universities which recommend long descriptions for complex images and diagrams still recommend the long desc instead of the aria details. So probably that's why probably not a lot of websites even know about this atribute. It is as always a matter of how the recommendations are communicated to universities and companies that reference aria specs, especially when things are deprecated and replaced by newer standards. See for example this website:
https://accessibility.asu.edu/articles/complex-images

So in short, aria details is replacing long desc atribute and is used mainly to describe complex images or diagrams, or tables, for which aria describtion or aria describedby is not enough.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details

**Comment by Adriani90** [comment]
I am in favor of keeping this settings but moving them to elements in the document formating settings because the aria details atribute is indeed a fully implemented Aria spec.

**Comment by Adriani90** [comment]
@CyrilleB79 aria details is supported in all browsers, long desc atribute was supported only in IE with the screen reader, in Firefox it is available only via the context menu but there is no suport for calling long desc from the element itself and the screen reader does not report if there is one. I think it is even not visible for sighted people at all.
https://www.w3.org/WAI/tutorials/images/complex/#a-text-link-to-the-long-description-adjacent-to-the-image
For Chrome there is a separate chrome extension for displaying long desc but I don't think it is widely used or known.

So in summary, we will very probably not be able to retrieve long desc information in every browser, and I don't think it is widely used at all. This long desc atribute has also been misused in the past, that was the main reason for deprecating it.
Most websites include the long description either on the page itself or provide a link to a table version or text version of data visualization next to the grafical view.
We will see if many websites wil adapt to this standard of aria details in the future. We could at least add a note in the user guide saying that NVDA supports this but there are not a lot of websites out there which implemented it yet.
